### PR TITLE
Readme updates new

### PR DESCRIPTION
**Pull request by reshmaserrao-awx**
@chao-ding-awx @shirly-chen-awx @carl-jin-awx 

Please review the updates to the airwallex-payment-elements Readme (JS Reference)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,503 +1,491 @@
...
+| `wechat`           | `intent`                 | Yes           | Intent                            | The `id` and `client_secret` of the PaymentIntent you want to checkout.  |
+| `applePayButton`   | `intent`                 | Yes           | Intent                            | The `id` and `client_secret` of the PaymentIntent you want to checkout. |
+|                    | `amount`                 | Yes           | {value: number; currency:string;} | The amount and currency of the PaymentIntent.    |
+|                    | `country_code`           | Yes           | string                            | Your two-letter ISO 3166 country code.    |
+|                    | `autoCapture`            | No            | boolean                           | Indicates whether the amount must be automatically captured after successful authorization. By default, the value from Create a PaymentIntent API will be used.   |
+|                    | `ApplePayRequestOptions` | No            | ApplePayRequestOptions            | Additional customization options for the button style and configuration for payment processing capabilities, billing and shipping information.    |
```
Comment (shirly-chen-awx) [inline docs/README.md:166]: can you help make up all the ApplePayRequestOptions here,  just copy the following and remove this line

| `buttonType`                   | No            | string                                 | The type of button you want to display on your payment form, for example, 'donate' https://developer.apple.com/documentation/apple\_pay\_on\_the\_web/displaying\_apple\_pay\_buttons\_using\_css |
| `buttonColor`                  | No            | string                                 | The color of the button. Default value is 'black'                                                                                                                                                 |
| `totalPriceLabel`              | No            | string                                 | The business name your shoppers will see when they look for the charge on their bank or credit card statement. For example, "COMPANY, INC."                                                       |
| `lineItems`                    | No            | ApplePayJS.ApplePayLineItem\[\];       | A set of line items that explain recurring payments and/or additional charges.                                                                                                                    |
| `billingContact`               | No            | ApplePayJS.ApplePayPaymentContact      | The billing contact information for the shopper.                                                                                                                                                  |
| `requiredBillingContactFields` | No            | ApplePayJS.ApplePayContactField\[\];   | The billing information that you require from the shopper to process the transaction.                                                                                                             |
| `shippingContact`              | No            | ApplePayJS.ApplePayPaymentContact      | Shipping contact information for the user.                                                                                                                                                        |
| `shippingMethods`              | No            | ApplePayJS.ApplePayShippingMethod\[\]; | The list of shipping methods available for a payment request.                                                                                                                                     |
| `shippingType`                 | No            | ApplePayJS.ApplePayShippingType;       | An optional value that indicates how purchased items are to be shipped.                                                                                                                           |
| `supportedCountries`           | No            | string\[ \]                            | A list of two-letter ISO 3166 country codes, used to limit payments to cards from specific countries or regions.                                                                                  |
| `applicationData`              | No            | string                                 | Optional user-defined data.

File: docs/README.md [hunk 2]
```diff
@@ -1,503 +1,491 @@
...
+## Interact with Embedded Elements
+
+### Create an Element
 
-### Element
+This method creates an instance of an individual Element. It takes the `type` of Element to create and an optional `options` object. Elements are rendered as iframes. 
 
-Functions and field components that can be used during checkout integration. Elements are rendered as iframes.
+```ts
+Airwallex.createElement(type, options)
```
Comment (shirly-chen-awx) [inline docs/README.md:102]: if customer is using npm, the code will be the following:
```
import { createElement } from '@airwallex/airwallex-payment-elements';
const element = createElement(type, options)
```

File: docs/README.md [hunk 3]
```diff
@@ -1,503 +1,491 @@
...
-| onBlur                      | The event is raised when an input in element loses focus.                                                                                                                                |
-| onChange                    | The events fire when the user commits a value change to a input. This may be done, for example, by clicking outside of the input or by using the Tab key to switch to a different input. |
-| onClick                     | The event is raised when the user clicks on an input element.                                                                                                                            |
-| onDynamicCurrencyConversion | The events fire when merchant enable Dynamic Currency Conversion (DCC) feature and shopper is confirm payment with an intent which match DCC scenario                                    |
+### Get an Element
+
+This method looks up a previously created Element by its type.  
+
+```ts
+Airwallex.getElement(type)
```
Comment (shirly-chen-awx) [inline docs/README.md:176]: same above, if customer is using npm, the code will be the following:

import { getElement } from '@airwallex/airwallex-payment-elements';
const element = getElement(type)

File: docs/README.md [hunk 4]
```diff
@@ -1,503 +1,491 @@
...
 
-```js
-// Enter your Payment Intent secret keys here
-const intent_id = 'replace-with-your-intent-id';
-const client_secret = 'replace-with-your-client-secret';
-```
+| **Property**    | **Required?** | **Type** | **Description**         |
+| --------------- | ------------- | -------- | ----------------------- |
+| `email`         | No            | string   | Shopper’s email         |
+| `first_name`    | No            | string   | Shopper’s first name    |
```
Comment (shirly-chen-awx) [inline docs/README.md:444]: some fields are required in billing
https://www.airwallex.com/docs/api?v=2022-02-16#/Payment_Acceptance/Payment_Intents/_api_v1_pa_payment_intents__id__confirm/post
image

File: docs/README.md [hunk 5]
```diff
@@ -1,503 +1,491 @@
...
-  | 'esun'
-  | 'permata_atm'
-  | 'boost'
-  | 'shopee_pay'
-  | 'paypal'
-  | 'payu'
-  | 'ovo'
-  | 'bitpay'
-  | 'truemoney';
+window.addEventListener('onReady', (event) => {
```
Comment (carl-jin-awx) [inline docs/README.md:275]:
```
const element = mount('root');
element.addEventListener('onReady', (event) => {
  // do sth here
});
```
It is a more recommended way.

File: docs/README.md (hunk 1 above)
Comment (reshmaserrao-awx) [reply docs/README.md:166]: Updated.

File: docs/README.md (hunk 2 above)
Comment (reshmaserrao-awx) [reply docs/README.md:102]: Updated.

File: docs/README.md (hunk 3 above)
Comment (reshmaserrao-awx) [reply docs/README.md:176]: Updated.

File: docs/README.md (hunk 4 above)
Comment (reshmaserrao-awx) [reply docs/README.md:444]: @shirly-chen-awx :  The Address object which is part of Billing object contains 
`city`        
`country_code`
`postcode`    
`state`       
`street`      

first_name, last_name are already part of the Billing object

File: docs/README.md (hunk 5 above)
Comment (reshmaserrao-awx) [reply docs/README.md:275]: Updated.

File: docs/README.md (hunk 4 above)
Comment (shirly-chen-awx) [reply docs/README.md:466]: I mean the second column, first_name and last_name should be required for billing
image

Comment (reshmaserrao-awx) [reply docs/README.md:466]: Thanks @shirly-chen-awx!  Should I make "Address" object 'Required' in the "Billing" object?

Comment (reshmaserrao-awx) [reply docs/README.md:466]: just to clarify,  I have made first_name, last_name Required in the billing object.  In the Address object, I have made country_code, street, city Required.  

I think Address object in Billing is conditional but we can leave it as optional.

**Review by shirly-chen-awx — APPROVED** [review]

File: docs/README.md (hunk 4 above)
Comment (shirly-chen-awx) [reply docs/README.md]: Let's follow the api documentation.

**Pull request merged**